bika/lims/content/calculation.py

```diff
@@ -245,8 +244,8 @@ def getCalculationDependants(self):
         calculation's Formula.
         """
         deps = []
-        for service_uid in self.service_backreferences:
-            service = get_object_by_uid(service_uid)
+        backrefs = get_backreferences(self, 'AnalysisServiceCalculation', as_objects=1)
```

Comment:
I would drop this `as_objects` completely and let the field always return UIDs. With a set of UIDs all caller functions can wake up the objects if they need it with `map(api.get_object_by_uid, get_backreferences(self, 'AnalysisServiceCalculation'))` easily